### ISSUE TITLE
learning rate for finetuning

### ISSUE THREAD

**Issue by krishnakanthnakkav2** [microsoft/analysing_pii_leakage#15]
Hi,

Can you please help where do I provide the learning rate for the optimiser? I don't see to find it in any of the config files.

Thanks

**Comment by s-zanella** [comment]
We used the default initial learning rate in Transformers, 5e-5, with a schedule that decreases it linearly to 0.
